	Declare copy constructor's source as const.
	Required by g++-4.3.2 for OpenSolaris x86_64.
	Reported by Apostolos Syropoulos <asyropoulos@gmail.com>.
	Patch proposed by Jonathan Kew <jfkthame@googlemail.com>.

diff -ur engine-2.3.1.orig/include/graphite/Segment.h engine-2.3.1/include/graphite/Segment.h
--- engine-2.3.1.orig/include/graphite/Segment.h	2009-01-29 09:33:19.000000000 +0100
+++ engine-2.3.1/include/graphite/Segment.h	2010-03-29 12:24:52.380352551 +0200
@@ -64,7 +64,7 @@
 	virtual ~Segment();
 
 	// Basic copy constructor:
-	Segment(Segment & seg);
+	Segment(const Segment & seg);
 
 	// For making modified copies of segments:
 	static Segment * LineContextSegment(Segment & seg, bool fStartLine, bool fEndLine);
diff -ur engine-2.3.1.orig/src/segment/Segment.cpp engine-2.3.1/src/segment/Segment.cpp
--- engine-2.3.1.orig/src/segment/Segment.cpp	2008-08-21 16:24:32.000000000 +0200
+++ engine-2.3.1/src/segment/Segment.cpp	2010-03-29 12:24:50.067202026 +0200
@@ -433,7 +433,7 @@
 /*----------------------------------------------------------------------------------------------
 	Basic copy method.
 ----------------------------------------------------------------------------------------------*/
-Segment::Segment(Segment & seg)
+Segment::Segment(const Segment & seg)
 {
 	int islout;
 
